content/2022-munich/health-and-safety.md

```diff
@@ -8,51 +8,14 @@ title: Submit Talk
 
 In the interest of keeping our events accessible to as much of the community as possible while still making sure our community is safe and protected, we have updated our Health + Safety policies for PromCon EU 2022.
 
-* All attendees must show proof of COVID-19 vaccination OR (1) negative COVID-19 test upon first entry to the event.
-* We will no longer conduct daily temperature checks upon entry for attendees.
-* <strong>Masks will still be required</strong> at all times unless actively eating or drinking, outdoors, or giving a keynote or breakout talk.
+* <strong>FFP2 Masks will be required</strong> at all times unless actively eating or drinking, outdoors, or giving a keynote or breakout talk.
 
 If you are traveling to Germany, review the entry requirements from your country of origin [here](https://www.auswaertiges-amt.de/en/coronavirus/2317268). As of Saturday, June 11, 2022, all COVID-19 entry restrictions to Germany will be provisionally lifted. From this date, entries to Germany will again be permitted for all travel purposes (including tourism and visiting trips). We highly suggest visiting the Germany Federal Foreign Office website regularly for the most up-to-date information as your trip nears. Changes and updates are typically marked at the top of the page. Read carefully to ensure you have the most recent information regarding travel rules and regulations based on your country of origin. The requirements may change so continue to check back regularly.
 
-**Note that entry to the event requires proof of vaccination or negative COVID-19 test, regardless of the Germany’s entry requirements.*
-
 ### Mask Mandate
 
 **While many municipalities’ mask mandates have been lifted masks will still be required to be worn at the event.**
 
-Masks must be worn over the nose and mouth and must be made of a tight-knit, non-permeable material. KN95 or N95 masks are recommended. Cloth masks are acceptable. Bandanas, scarves, and gators are not permitted to be used as masks.
+Masks must be worn over the nose and mouth and must be made of a tight-knit, non-permeable material. KN95, N95, or FFP2 masks are required.
 
 Masks do not need to be worn when in outdoor spaces and while actively eating food and beverage indoors when socially distanced from others. Speakers are not required to wear masks while presenting.
-
```

Comment:
I mean, the CNCF (Elna) asked us to add all of this. If we remove it and they are legally the entity behind it, we should at least check with them.